features/admin_interacts_with_dashboard.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Admin views and edits buildings on dashboard
+  Scenario: Admin can see relevant content
+    Given I am on the dashboard
+    Then the dashboard contains all expected columns
+    And the dashboard contains the expected building information
```

Comment:
I find this line quite vague so I'd actually remove the step "`And(the dashboard contains[...]`) and put the expectations directly in here.

Reply:
Done!